internal/signalfx-agent/pkg/monitors/prometheusexporter/prometheus.go

```diff
@@ -133,30 +134,37 @@ func (m *Monitor) Configure(conf *Config) error {
 
 	var ctx context.Context
 	ctx, m.cancel = context.WithCancel(context.Background())
+	applyDimensions := func(attrs pcommon.Map) {
+		attrs.PutStr("system.type", m.monitorName)
+		for k, v := range conf.ExtraDimensions {
+			attrs.PutStr(k, v)
+		}
+	}
```

Comment:
Mind shifting this to smartagentreceiver/output.go so all monitors pick it up for free?